bofire/data_models/surrogates/enting.py

```diff
@@ -0,0 +1,27 @@
+from typing import Literal
+
+from pydantic import Field
+from typing_extensions import Annotated
+
+from bofire.data_models.surrogates.surrogate import Surrogate
+from bofire.data_models.surrogates.trainable import TrainableSurrogate
+
+
+class EntingSurrogate(Surrogate, TrainableSurrogate):
```

Comment:
If you want to do it like this, I would call it `LGBMEntingSurrogate`, and if you have xgb support at some point you could introduce an `XGBEntingSurrogate`. This has the advantage that the names of hyperparameters do not have to be the same but different, or if one hyperparam is only available in one model you only provide it in this model.

Reply:
The other option would be to have a `LGBMSurrogate` which just includes the mean model and implement `LGBMEntingSurrogate` as inherited class from `LGBMSurrogate` and add the uncertainty parameters there. But I know that in the current `Entmoot` model it is setup a bit different, so you would have in this case two different `_fit` methods. The first just uses LGBM and the second one ENTMOOT. 

Thinking this further one could even build it up by having an `LGBMSurrogate` and `XGBSurrogate` class which are ENTMOOT agnostic and all care for the mean model and an `EntingUncertaintySurrogate` which adds the uncertainty part to the model. Then one could build up the `LGBMEntingSurrogate` by inheritance from `LGBMSurrogate` and `EntingUncertaintySurrogate`. This would be the super object oriented solution, but as these kind of structures are currently not supported by ENTMOOT, I think its is ok to just go with the flat `LGBMEntingSurrogate`.

What do you think?